docs/en/ingest-management/elastic-agent/elastic-agent-unprivileged-mode.asciidoc

```diff
@@ -0,0 +1,185 @@
+
+// Add mention of the System integration data streams.
+
+You can also <<unprivileged-change-mode,change the privilege mode>> of an {agent} after it has been installed.
+
+Refer to <<unprivileged-command-behaviors>> and <<unprivileged-running>> for the requirements and steps associated with running an agent without full `root` or `admin` superuser privileges.
+
+* <<unprivileged-running>>
+* <<unprivileged-command-behaviors>>
+* <<unprivileged-integrations>>
+* <<unprivileged-view-mode>>
+* <<unprivileged-change-mode>>
+
+[discrete]
+[[unprivileged-running]]
+== Run {agent} in `unprivileged` mode
+
+To run {agent} without administrative privileges you use exactly the same commands that you use for {agent} otherwise, with one exception. When you run the <<elastic-agent-install-command,`elastic-agent install`>> command, add the `--unprivileged` flag. For example:
+
+[source,shell]
+----
+elastic-agent install \
+  --url=https://cedd4e0e21e240b4s2bbbebdf1d6d52f.fleet.eu-west-1.aws.cld.elstc.co:443 \
+  --enrollment-token=NEFmVllaa0JLRXhKebVKVTR5TTI6N2JaVlJpSGpScmV0ZUVnZVlRUExFQQ== \
+  --unprivileged
+----
+
+IMPORTANT: On Linux systems, when you install {agent} using the `--unprivileged` flag, {agent} commands should not be run with `sudo`.
+Doing so may result in <<agent-sudo-error,an error>> due to the agent not having the required privileges.
+
+[discrete]
+[[unprivileged-command-behaviors]]
+== Agent and dashboard behaviors in unprivileged mode
+
+In addition to the <<unprivileged-integrations,integrations that are not available>> when {agent} is run in unpriviledged mode, certain data streams are also not available. The following tables show, for different operating systems, the impact when the agent does not have full administrative privileges. In most cases the limitations can be mediated by granting permissions for a user or group to the files indicated.
+
+.macOS
+[options,header]
+|===
+|Action |Behavior in unprivileged mode |Resolution
+
+|Run {agent} with the System integration
+|Log file error: `Unexpected file opening error: Failed opening /var/log/system.log: open /var/log/system.log: permission denied`.
+|Give read permission to the `elastic-agent` group for the `/var/log/system.log` file to fix this error.
+
+|Run {agent} with the System integration
+|On the `[Logs System] Syslog` dashboard, the `Syslog events by hostname`, `Syslog hostnames and processes` and `Syslog logs` visualizations are are missing data.
+|Give read permission to the `elastic-agent` group for the `/var/log/system.log` file to fix the missing visualizations.
+
+|Run {agent} with the System integration
+|On the `[Metrics System] Host overview` dashboard, only the processes ran by the `elastic-agent-user` user are shown in the CPU and memory usage lists.
+|Give read permission to the `elastic-agent` group for the `/var/log/system.log` file to fix the missing visualizations.
```

Comment:
I'm not sure this is the right resolution for this particular symptom. @blakerouse Please keep me honest but  I don't think there is any resolution here (short of running Agent as `root`, which we can't do in `unprivileged` mode by definition).

Reply:
Ah, my mistake! Kaan's notes say only `This confirms that only the processes ran by the elastic-agent-user user are shown in the cpu and memory usage list.` And I see the resolution I added is for a different issue, specifically that on the `[Elastic Agent] Agent Info` dashboard,  the `[Elastic Agent] Most Active Agents` visualization isn't populated. I'll fix this.

Reply:
Not your mistake.  I went back and looked in https://github.com/elastic/elastic-agent/issues/4705 and it has the same incorrect resolutions too — I think it was just a copy/paste error. 🙂

Reply:
An option on macOS is to add the `elastic-agent-user` to the `admin` group. That is probably preferred from changing the ownership of that file, but possible that it gives the `elastic-agent-user` more permissions than the user wants.

Reply:
I've changed this as shown:
![Screenshot 2024-06-25 at 11 05 42 AM](https://github.com/elastic/ingest-docs/assets/41695641/6edfbbd8-cc9b-4344-99a1-c6380542b37d)

Reply:
That looks really good to me, @kilfoyle!

Reply:
BTW, these same fixes need to be applied to the corresponding items for Linux.

Reply:
I've updated three Linux `Run {agent} with the System integration` items as shown:

![Screenshot 2024-06-25 at 2 04 02 PM](https://github.com/elastic/ingest-docs/assets/41695641/614303b1-13b6-4bfd-a8f5-013fce921dc8)

---

The two Linux `Run {agent} and access the {agent} dashboards` items are unchanged:

![screen](https://github.com/elastic/ingest-docs/assets/41695641/cb46b077-aed2-4025-9d0b-4d58a11a2f67)